afterpay/src/main/assets/widget/index.html

```diff
@@ -0,0 +1,15 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+    <meta content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no" name="viewport">
+    <title>Afterpay Payment Schedule</title>
+    <script src="%s"></script>
+    <script src="%s"></script>
+</head>
+
+<body>
+<div id="afterpay-widget-container" style="margin: 16px 0px"></div>
```

Comment:
It's actually good that we have the HTML embedded in the apps now. It means you can adjust the style here in future if you need to.